test/github-scan

```diff
@@ -27,12 +27,14 @@ def main():
     parser = argparse.ArgumentParser(description='Test infrastructure: scan and update status of pull requests on github')
     parser.add_argument('-d', '--dry', action="store_true", default=False,
                         help='Don''t actually change anything on github')
-    parser.add_argument('image', action='store', nargs='?',
-                        help='The operating system image to scan against')
+    parser.add_argument('--except', dest='without', action='store_true',
+                        help='Scan for anything except the specified context')
+    parser.add_argument('context', action='store', nargs='?',
```

Comment:
In the current implementation, `--except` and `context` contradict each other (`--except` overrides), yet this isn't tested here.
I think it would make more sense to obviously make them contradict in the help and to allow multiple `--except` entries. Possible hint for `context`: 'Scan only this context for tasks'